mods/ctf/ctf_treasure/init.lua

```diff
@@ -23,7 +23,7 @@ function ctf_treasure.get_default_treasures()
 		{"shooter:ammo",0.3,2,{1,10}},
 		{"shooter:arrow_white",0.5,2,{2,18}},
 
-		{"ctf_bandages:bandage",0.3,5,{1,6}},
+		{"ctf_bandages:bandage",0.85,5,{2,4}},
```

Comment:
```suggestion
		{"ctf_bandages:bandage",0.8,2,{2,4}},
```

I've tested this, and there are a lot of bandages being spawned. The spawn rate is also a tad higher than medkits, as it should be.